Vue/vaahtwo/stores/root.js

```diff
@@ -118,7 +118,7 @@ export const useRootStore = defineStore({
         {
             if(data && data.is_logged_in == false)
             {
-                this.$router.push({name: 'sign.in'})
+                window.location.href = this.base_url+'#';
```

Comment:
@we-vikram-wri231 test it properly, try to host vaahcms on a domain in local machine like how we do it in oho.